HW_Python/Expl2.py

```diff
@@ -0,0 +1,14 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+n = int(input("Введите число журавликов: "))
+x = int(((n/3)/2))
```

Comment:
Приоритет у операции деления одинаковый, значит, скобки не нужны. Функция `int` тут также не нужна, так как существует отличный оператор целочисленного отделения.
```suggestion
x = n // 3 // 2
```